test/utils/BaseVm.sol

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity >=0.8.22;
+
+import { Vm } from "forge-std/src/Vm.sol";
+
+abstract contract BaseVm {
```

Comment:
Can we add a TODO comment to remove this once they implement it [upstream](https://github.com/foundry-rs/forge-std/issues/530)?